modules/components/pages/inputs/schema_registry.adoc

```diff
@@ -0,0 +1,285 @@
+= schema_registry
+:type: input
+:status: beta
+:categories: ["Integration"]
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Reads schemas from a schema registry. You can use this connector to extract and backup schemas during a data migration.
```

Comment:
we usually use title case for Schema Registry but that's probably when it's the one embedded in Redpanda. I assume this is lowercase here because it's platform agnostic.

Reply:
yes, that's right.